Resources/Prototypes/_Goobstation/Entities/Wallmounts/Signs/posters.yml

```
@@ -0,0 +1,100 @@
+# Posters
+
+- type: entity
+  parent: PosterBase
+  id: PosterLegitGoobWorkHard
+  name: "Work Hard"
+  description: "If you work hard enough, YOU could earn your very own Nanotrasen Plunger Plus! Just be sure to survive long enough to redeem it!.."
+  components:
+  - type: Sprite
```

yamllint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)